src/settings.ts

```diff
@@ -553,6 +555,7 @@ export const FIREBASE_CONFIG = {
   measurementId: MEASUREMENT_ID,
   projectId: PROJECT_ID,
   storageBucket: BUCKET_URL,
+  authDomain: AUTH_DOMAIN,
```

Comment:
The rest of the keys here seem to be sorted alphabetically. Would be good to maintain that.

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/1546/commits/c5f8848eb58b5921d9ff5785eba01ee6b0bb2278